ClosedXML/Excel/NamedRanges/XLNamedRange.cs

```diff
@@ -15,7 +15,7 @@ internal class XLNamedRange : IXLNamedRange
         public XLNamedRange(XLNamedRanges namedRanges, String rangeName, String range, String comment = null)
             : this(namedRanges, rangeName, comment)
         {
-            RangeList.Add(range);
+            range.Split(',').ForEach(r => RangeList.Add(r));
```

Comment:
Just note that this implementation is quite naive, because a sheet name can contain a comma. But we do `Split(',')` in many places in this class. Once we implement a proper named range parser we can work on this.

Reply:
Will add a TODO